src/components/CustomClient.js

```diff
@@ -11,10 +11,13 @@ import { Form, Message } from 'semantic-ui-react';
  */
 
 /**
- * Accept data from a JSON object and send it back.
+ * Accept data from a user's JSON object and send it to `load`.
+ *     Currently for development only.
  * 
  * @param {object} props
- * @param {load} props.load
+ * @param {toLoad} props.load
```

Comment:
iirc, there's a  type definition above that should allow this to work.

Reply:
Ah cool, didn't know you could do that!
I think the type should just be `load` though in that case, since that's the name of the callback. `toLoad` is the object that's passed to `load`. 

(Although actually `toLoad` is a sub-object of the parameter that's passed, so we should really change the callback definition to have `params.toLoad` instead of just `toLoad`.)

Reply:
OK, I don't see it. The comment above says `@callback load` - doesn't that mean that `load` is the name of the callback type?

Also, this is unrelated, but why doesn't this class have a constructor?

Reply:
Ok, that should be a bit better. You were right, it was completely mixed up.

Reply:
@ethanbb The class doesn't have a constructor because there's nothing the constructor would do and our linting rules don't like useless constructors. It seems like there's probably a default constructor that just instantiates the class with `super(props)` or something.

Ready for merging?